src/components/Dashboard/Dashboard.js

```diff
@@ -32,6 +36,10 @@ class Dashboard extends React.Component {
   };
 
   render() {
+    if (this.state.toBookDetail) {
```

Comment:
why do you need this? here?

Reply:
I changed it in the next PR to using Route